x-pack/plugins/cross_cluster_replication/public/app/components/form_entry_row.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { debounce } from 'lodash';
+
+import {
+  EuiTitle,
+  EuiFieldText,
+  EuiFieldNumber,
+  EuiDescribedFormGroup,
+  EuiFormRow,
+} from '@elastic/eui';
+
+import { i18nValidationErrorMessages } from '../services/input_validation';
+
+/**
+ * State transitions: fields update
+ */
+export const updateFields = (newValues) => ({ fields }) => ({
+  fields: {
+    ...fields,
+    ...newValues,
+  },
+});
+
+const parseError = (err) => {
+  if (!err) {
+    return null;
+  }
+
+  const [error] = err.details; // Use the first error in the details array (error.details[0])
```

Comment:
Minor nit: this could be rewritten the way the comment describes it, which would make the comment unnecessary. 😄 

```js
const error = err.details[0];
```